src/Mollie/WC/Gateway/Creditcard.php

```diff
@@ -101,6 +113,82 @@ protected function includeCreditCardIconSelector()
             '/inc/settings/mollie_creditcard_icons_selector.php'
         );
 
-        $this->form_fields = array_merge($this->form_fields, $fields);
+        $fields and $this->form_fields = array_merge($this->form_fields, $fields);
+    }
+
+    /**
+     * @return array Array containing the credit cards names enabled in settings
+     *               to make customization of checkout icons
+     */
+    protected function enabledCreditcards()
+    {
+        $creditCardSettings = get_option('mollie_wc_gateway_creditcard_settings', false) ?: [];
+        $enabled = isset($creditCardSettings[Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS_ENABLER])
+            ? wc_string_to_bool($creditCardSettings[Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS_ENABLER])
+            : false;
+
+        if (!$enabled) {
+            return [];
+        }
+        $optionLexem = Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS;
+        $creditcardsAvailable = Mollie_WC_Helper_PaymentMethodsIconUrl::AVAILABLE_CREDITCARD_ICONS;
+        $svgFileName = Mollie_WC_Helper_PaymentMethodsIconUrl::SVG_FILE_EXTENSION;
```

Comment:
I would move this on top of the method + I would assign to variable the constant above `Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS_ENABLER` too. Will be more clear